api/routes.go

```diff
@@ -46,6 +48,21 @@ func InstallRoutes(r *mux.Router, sdkRouter *sdkrouter.Router) {
 	v1Router.HandleFunc("/proxy", proxy.Handle).Methods(http.MethodPost)
 	v1Router.HandleFunc("/proxy", emptyHandler).Methods(http.MethodOptions)
 
+	tusHandler, err := publish.NewHandler(uploadPath)
+	if err != nil {
+		logger.Log().WithError(err).Fatal(err)
+	}
+
+	tusRouter := v1Router.PathPrefix("/files").Subrouter()
```

Comment:
This probably should live under v2, with old handler living in the v1 alongside.

Reply:
gotcha